ui/main/src/app/business/view/userCard/cardBuilder/cardBuilder.ts

```diff
@@ -308,11 +308,11 @@ export class CardBuilder {
             );
         if (this.specificCardInformation.timeSpans) {
             return this.specificCardInformation.timeSpans?.map((timeSpan) => {
-                return new TimeSpan(timeSpan.startDate, timeSpan.endDate, timeSpan.recurrence);
+                return new TimeSpan(timeSpan.startDate, timeSpan.endDate);
             });
         }
         if (this.specificCardInformation.viewCardInCalendar) {
-            return [new TimeSpan(this.startDate, this.endDate, this.specificCardInformation.recurrence)];
+            return [new TimeSpan(this.startDate, this.endDate)];
         }
```

Comment:
To remove (lines below) 
```
if (this.specificCardInformation.recurrence)
            logger.warn(
                "Using deprecated field 'specificInformation.recurrence'. Use 'specificInformation.timeSpan' field instead to configure timespans"
            );

```

This appears in two places in the file : line 154 and 307